.travis.yml

```diff
@@ -8,7 +8,7 @@ cache:
 
 matrix:
   include:
-    - php: 5.5
```

Comment:
we should change the travis file to check also lowest composer dependencies.

```
matrix:
  include:
    - php: 5.5
      env:
        - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
    - php: 7.0
      env:
        - COMPOSER_FLAGS="--prefer-dist --no-interaction"

before_script:
      - composer update $COMPOSER_FLAGS
```

This will test the lowest and the highest deps.